src/app/templates/registration/password_reset_confirm.html

```diff
@@ -0,0 +1,42 @@
+{% extends 'registration/base_site.html' %}
+
+{% block content %}
+  {% if validlink %}
+      <div class="col-md-3">
+      </div>
+      <div class="col-md-6">
+      <div class="panel panel-default">
+      <div class="panel-body">
+        <form id="password_confirm_form" role="form"  class="form-horizontal"  method="post" enctype="multipart/form-data">
+          {% csrf_token %}
+          {{ form.new_password2.errors }}
+          <div class="form-group">
+            <label class="control-label col-sm-4" for="new_password1">New password:</label>
+            	<div class="col-sm-8">
+              		{{ form.new_password1 }}
+            	</div>
+          </div>
+          <div class="left-align">
+            {{ form.new_password1.help_text|safe }}
+          </div>
+          {{ form.new_password2.errors }}
+          <div class="form-group">
+            <label class="control-label col-sm-4" for="new_password2">New password confirmation:</label>
+            	<div class="col-sm-8">
+              		{{ form.new_password2 }}
+            	</div>
+          </div>
+          <button type="submit">Change password</button>
+        </form>
+      </div>
+      </div>
+      </div>
+      <div class="col-md-3">
+      </div>
+  {% else %}
+    <p>
+      The password reset link was invalid, possibly because it has already been used.
+      Please request a new password reset.
+    </p>
+  {% endif %}
+{% endblock %}
```

Comment:
![screenshot from 2018-02-05 20-43-52](https://user-images.githubusercontent.com/17159160/35811848-8899e990-0ab5-11e8-91f3-ead02495fce7.png)

Shift the help text above the two form fields.

Reply:
Done